test/utils.js

```diff
@@ -35,13 +35,16 @@ const FIREFOX_PREFERENCES = {
   // NECESSARY for all 57+ builds
   "extensions.legacy.enabled": true,
 
+  // Force variation for testing
+  "extensions.button_icon_preference.variation": "kittens",
```

Comment:
If we're going to force the variation to "kittens", we should add a note in `DEV.md` and `TESTPLAN.md` that this pref can be used to force a variation (and similarly, removed from here to see how `weightedVariation` works from `Config.jsm` to select the treatment each time).

There should probably also be a comment in `bootstrap.js` where this pref is defined to explain this a bit more in the code.

Reply:
Good suggestions. Done